editor/components/rich-text/index.js

```diff
@@ -830,13 +830,7 @@ export class RichText extends Component {
 			!! this.editor &&
 			this.props.tagName === prevProps.tagName &&
 			this.props.value !== prevProps.value &&
-			this.props.value !== this.savedContent &&
-
-			// Comparing using isEqual is necessary especially to avoid unnecessary updateContent calls
-			// This fixes issues in multi richText blocks like quotes when moving the focus between
-			// the different editables.
-			! isEqual( this.props.value, prevProps.value ) &&
-			! isEqual( this.props.value, this.savedContent )
```

Comment:
Would there be any point in keeping the check to log a warning in dev mode if values are the same but an update is triggered?

Reply:
>Would there be any point in keeping the check to log a warning in dev mode if values are the same but an update is triggered?

I love that idea!